src/prefs/SpectrogramSettings.cpp

```diff
@@ -254,7 +272,7 @@ void SpectrogramSettings::LoadPrefs()
 
    gPrefs->Read(wxT("/Spectrum/WindowType"), &windowType, eWinFuncHann);
 
-   isGrayscale = (gPrefs->Read(wxT("/Spectrum/Grayscale"), 0L) != 0);
+   colorScheme = ColorScheme(gPrefs->Read(wxT("/Spectrum/ColorScheme"), 0L));
```

Comment:
Spectrogram has per-track setting. Can it really be static? I'm confused.

Reply:
The old code here is in fact using a global setting.

The system of spectrogram settings leaves something to be desired -- a track can have independent settings, or it can take its settings from defaults, but it can't default individual settings separately.

Don't try to understand and fix all of that -- just understand that where you rewrite usages of a global setting (which are what gPrefs contains), then it is preferable to use an EnumSetting object instead.

Reply:
Done.